sycl/test-e2e/MultiDevice/set_arg_pointer.cpp

```diff
@@ -0,0 +1,63 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// UNSUPPORTED: level_zero_v2_adapter
```

Comment:
This should work on v2 by now. Should probably be changed in the sycl branch, not here.

Reply:
PR: https://github.com/intel/llvm/pull/20913/changes